exonum/src/blockchain/config.rs

```diff
@@ -46,6 +46,9 @@ pub struct StoredConfiguration {
     pub validator_keys: Vec<ValidatorKeys>,
     /// Consensus algorithm parameters.
     pub consensus: ConsensusConfig,
+    /// Number of votes to commit configuration
+    /// Should be greater or equal to 2/3 of validators count
```

Comment:
...greater or equal to 2/3 + 1 or `+2/3`.
Additionally it is worth mentioning that this value should be less or equal to the validators number.